mlflow_faculty/trackingstore.py

```diff
@@ -199,7 +201,21 @@ def create_run(
 
         :return: The created Run object
         """
-        raise NotImplementedError()
+        try:
+            faculty_run = self._client.create_run(
+                self._project_id,
+                experiment_id,
+                datetime.fromtimestamp(start_time / 1000, tz=UTC),
```

Comment:
Did you consider putting this logic as a helper in the converters for clear symmetry with the opposite conversion logic?

Reply:
done